Cabal/src/Distribution/Simple/BuildPaths.hs

```diff
@@ -170,7 +172,19 @@ getSourceFiles verbosity dirs modules = flip traverse modules $ \m -> fmap ((,)
     findFileWithExtension ["hs", "lhs", "hsig", "lhsig"] dirs (ModuleName.toFilePath m)
       >>= maybe (notFound m) (return . normalise)
   where
-    notFound module_ = die' verbosity $ "can't find source for module " ++ prettyShow module_
+    notFound :: ModuleName.ModuleName -> IO FilePath
+    notFound module_ = do
+      possibleFilePaths <- concat <$> (traverse getDirectoryContentsRecursive dirs)
```

Comment:
I suspect parentheses are redundant (you can check with `hlint` if you want)

Reply:
I think this is another thing that'd be fixed if I had HLS working, maybe?